data/resources/plugins/renameitem.py

```
@@ -61,7 +67,49 @@ def callback(self, *args):
-        rename.set_data(doc)
-        actions.show_stack_page_by_name('rename')
+        srvdlg = self.app.get_service('dialogs')
+        # ~ actions = self.app.get_service('actions')
+        rename_widget = self.app.add_widget('rename-widget', MiAZRenameDialog(self.app))
+        rename_widget.set_data(doc)
+        dtype = "question"
+        text = '' # _(f'<big>{i_title} {selected_item.id} is still being used</big>')
+        window = self.app.get_widget('window')
+        title = "Rename document"
+        dialog = srvdlg.create(enable_response=True, dtype=dtype, title=title, body=text, widget=rename_widget)
+        dialog.add_response("preview", _("Preview"))
```

Ruff (0.8.2)

Undefined name `_`

(F821)